linux_os/guide/system/software/updating/dnf-automatic_security_updates_only/oval/shared.xml

```diff
@@ -0,0 +1,12 @@
+{{{
+oval_check_ini_file(
```

Comment:
Please try to skip default values. And change the macro to use 'missing_config_file_fail=true` as default which is probably the most common case then you don't need to specify every time this parameter.

Also depending on how much `"/etc/dnf/automatic.conf"` will be changed by other rules I would create a macro to encapsulate that.

Reply:
Great. Thanks for changing that.